targets/targets.json

```diff
@@ -4188,12 +4188,15 @@
         "overrides": {
             "lse_available": 0
         },
-        "macros_add": ["USB_STM_HAL", "USBHOST_OTHER"],
         "device_has_add": [
-            "ANALOGOUT",
+            "TRNG",
             "MPU"
         ],
-        "device_has_remove": ["LPTICKER"],
+        "device_has_remove": [
+			"LPTICKER",
```

Comment:
Can you please align these lines, same as MPU above (just 4 spaces to right)

Reply:
+it's using tabs instead of spaces

Reply:
Thanks for catching that. Fixed now. @0xc0170 @kjbracey-arm